### PR TITLE
MGT 2.0 Release Docs Update

### PR DESCRIPTION
**Pull request by elisenyang**
Docs updates to go with MGT 2.0 release on Dec 10th.

### CODE REVIEW

File: concepts/toc.yml [hunk 1]
```diff
@@ -553,18 +553,36 @@ items:
...
+              - name: Build a web app (JavaScript)
                 href: toolkit/get-started/build-a-web-app.md
+              - name: Build a web app (React)
+                href: toolkit/get-started/use-toolkit-with-react.md
+              - name: Build a web app (Angular)
+                href: toolkit/get-started/use-toolkit-with-angular.md
               - name: Build a SharePoint web part
                 href: toolkit/get-started/build-a-sharepoint-web-part.md
-              - name: Build a Microsoft Teams tab
+              - name: Build a Microsoft Teams Tab
```
Comment (Lauragra) [inline concepts/toc.yml:570]: The name for this technology in Microsoft Teams is not treated as a proper noun and capitalized; we should lowercase it here.

File: concepts/toc.yml [hunk 2]
```diff
@@ -585,24 +603,20 @@ items:
...
-            href: toolkit/templates.md
+              - name: Styling components
+                href: toolkit/customize-components/style.md
+              - name: Templating components
+                href: toolkit/customize-components/templates.md
+              - name: Caching Microsoft Graph calls in components
+                href: toolkit/customize-components/cache.md
+              - name: Localizing components
+                href: toolkit/customize-components/localization.md
+              - name: Right-To-Left
```
Comment (Lauragra) [inline concepts/toc.yml:618]: I suggest using "Right-to-left support" for the TOC entry here.

File: concepts/toc.yml (hunk 1 above)
Comment (elisenyang) [reply concepts/toc.yml]: @Lauragra Updated this in the TOC to lowercase

File: concepts/toc.yml (hunk 2 above)
Comment (elisenyang) [reply concepts/toc.yml]: Updated this in the TOC

File: concepts/toolkit/components/todo.md [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+| Event | Detail | Description |
+| --- | --- | --- |
+| taskAdded | The detail contains the respective `task` object | Fires when a new task has been created. |
+| taskChanged | The detail contains the respective `task` object | Fires when task metadata has been changed, such as marking completed. |
+| taskClick | The detail contains the respective `task` object | Fires when the user clicks or taps on a task. |
+| taskRemoved | The detail contains the respective `task` object | Fires when an existing task has been deleted. |
+
+## Templates
+
+The `tasks` component supports several [templates](../customize-components/templates.md) that allow you to replace certain parts of the component. To specify a template, include a `<template>` element inside a component and set the `data-type` value to one of the following:
```
Comment (Lauragra) [inline concepts/toolkit/components/todo.md]: We should be referring to the to-do component here, right?

File: concepts/toolkit/components/todo.md [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+        <div>
+            Importance Level: {{task.importance}}
+        </div>
+    </template>
+</mgt-todo>
+```
+
+## Microsoft Graph permissions
+
+This control uses the following Microsoft Graph APIs and permissions.
```
Comment (Lauragra) [inline concepts/toolkit/components/todo.md:127]: In this context "This control" is the to-do component?

File: concepts/toolkit/components/todo.md [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+
+| Resource | Permission |
+| - | - |
+| [/me/todo/lists/](/graph/api/todo-list-lists) | Tasks.ReadWrite |
+| [/me/todo/lists/{todoTaskListId}/tasks](/graph/api/todotasklist-list-tasks) | Tasks.ReadWrite |
+| [/me/todo/lists/{todoTaskListId}/tasks/{taskId}](/graph/api/todotask-get) | Tasks.ReadWrite |
+
+## Authentication
+
+The tasks component uses the global authentication provider described in the [authentication documentation](../providers/providers.md).
```
Comment (Lauragra) [inline concepts/toolkit/components/todo.md:137]: tasks or to-do?

File: concepts/toolkit/components/todo.md (hunk 3 above)
Comment (elisenyang) [reply concepts/toolkit/components/todo.md]: yes, changed it to To Do

File: concepts/toolkit/components/todo.md (hunk 4 above)
Comment (elisenyang) [reply concepts/toolkit/components/todo.md:127]: yes, updated this to be "To Do component"

File: concepts/toolkit/components/todo.md (hunk 5 above)
Comment (elisenyang) [reply concepts/toolkit/components/todo.md:137]: Updated to To Do

File: concepts/toolkit/customize-components/style.md [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+<mgt-people-picker class="mgt-dark custom-class"></mgt-people-picker>
+```
+```css
+mgt-people-picker.custom-class {
+    --input-background-color: $custom-background-color;
+    --input-border: $custom-input-border;
+}
+```
+
+## Disable the shadow dom (experimental)
```
Comment (Lauragra) [inline concepts/toolkit/customize-components/style.md:70]: What do we mean by calling this experimental?

Comment (nmetulev) [reply concepts/toolkit/customize-components/style.md:70]: This needs to be removed, I think we might have restored it in a merge when we moved the style docs. I'll delete this section

File: concepts/toolkit/customize-components/style.md [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+
+
+```html
+<script type="module">
+  import { MgtBaseComponent } from './dist/es6/components/baseComponent.js';
+   MgtBaseComponent.useShadowRoot = false;
+</script>
+
+<script type="module" src="./dist/es6/components/mgt-tasks/mgt-tasks.js"></script>
+````
```
Comment (nmetulev) [inline concepts/toolkit/customize-components/style.md:82]:
```suggestion
```

**Review by Lauragra — APPROVED** [review]

**Pull request merged**